app/src/main/java/pl/org/akai/eater/viewmodel/AuthViewModel.kt

```diff
@@ -0,0 +1,24 @@
+package pl.org.akai.eater.viewmodel
```

Comment:
Please keep code in packages named by features, so for example authscreen and authviewmodel will be together in package auth etc. THose packages can be in ui package.